frontend/src/Pages/RoomSettings.js

```diff
@@ -45,18 +50,15 @@ export default function RoomSettings() {
 
   useEffect( () => {
     async function fetchRoomData() {
-      const resp = await axios.get('api/users/room-list');
+      const resp = await axios.get('/api/users/room-list');
```

Comment:
Might be worth handling the error case in case this API fails (backend goes down). At the moment your fetchRoomData will just bleed the error out and the UI won't show anything.

Reply:
I've added a toast message popup in the Axios response interceptor which will be triggered every time the API call receives 401 or 502 status codes.

You can find it in this commit [ed4e5d7](https://github.com/MatrixDoctors/matrix-cerberus/pull/46/commits/ed4e5d73b0b0fb8fc173249047abd6af16e6bd85)